app/views/_includes/summary-cards/competencies.html

```diff
@@ -0,0 +1,50 @@
+{% set industryHtml %}
+  {# Build an list of the items selected #}
+  <ul class="govuk-list">
+    {% for role in data.industryExpertiseType %}
+      <li>
+        {% if role == "An option that isn't listed" %}
+          {{ data.industryExpertiseTypeOther }}
+        {% else %} 
+          {{ role }}
+        {% endif %}
+      </li>
+    {% endfor %}
+  </ul>
+{% endset %} 
+
+{{ xGovukSummaryCard({
+  
+  rows: [
+    {
+      key: {
+        text: "Provide details of your communication, judgement and analytical skills"
+      },
+      value: { 
+        text: data.competenciesDetails
```

Comment:
this data name needs to be unique. e.g. use `data.acheivementDetails` and `data.communicationDetails`